docs/guide/dependencies.md

```diff
@@ -9,7 +9,6 @@ features.
 | [bessel](https://github.com/SheetJS/bessel) | Apache v2.0 | SheetJS |
 | [Chevrotain](https://github.com/SAP/chevrotain) | Apache v2.0 | SAP SE or an SAP affiliate company |
 | [core-js](https://github.com/zloirock/core-js) | The MIT License | Denis Pushkarev |
-| [GPU.js](https://github.com/gpujs/gpu.js/) ([optional](enabling-gpu-acceleration.md)) | The MIT License | GPU.js team |
```

Comment:
Dependency is not yet removed. It's optional until v2.0

Reply:
Done: e8e8323